### PR TITLE
Board page

### PR DESCRIPTION
**Pull request by whirligigY**

### CODE REVIEW

File: src/components/Boards/Boards.jsx [hunk 1]
```diff
@@ -23,9 +27,21 @@ const Boards = () => {
...
+          >
+            <div>
+              <p>{item.title}</p>
+            </div>
+          </Link>
+=======
           <div key={item.id} className="board__list__board card">
             <p>{item.title}</p>
           </div>
+>>>>>>> origin/main
```
Comment (IgorLap239) [inline src/components/Boards/Boards.jsx:44]: need to remove extra lines after merge

File: src/utils.js [hunk 2]
```diff
@@ -26,27 +26,40 @@ export const useInput = (initial) => {
...
   };
-  const dragOverBoardHandler = (e) => {
+  const dragOverBoardHandler = (e, newOrder) => {
     e.preventDefault();
   };
 
-  const dragEndBoardHandler = (e) => {
-    e.target.style.opacity = '1';
+  const dragEndBoardHandler = (e, newOrder) => {
+    console.log(e.target);
```
Comment (IgorLap239) [inline src/utils.js:51]: stash console.log

**Review by IgorLap239 — APPROVED** [review]

File: src/components/Boards/Boards.jsx [hunk 3]
```diff
@@ -1,6 +1,10 @@
 import { Row } from "react-bootstrap";
 import { useState, useEffect } from "react";
+<<<<<<< HEAD
+import { Link } from "react-router-dom";
+=======
 import "./boards.css";
+>>>>>>> origin/main
```
Comment (Taneros) [inline src/components/Boards/Boards.jsx:8]: resolve convlict

File: src/components/Header/index.jsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  <header className="header">
+    <nav className="header__nav nav__button">
+      <button className="nav_button__more"></button>
+      <Link className="nav_button__trello" aria-current="page" to="/">
+        Crello
+      </Link>
+      <Link className="nav_button__trello" aria-current="page" to="/dashboard">
+        Dashboard
+      </Link>
+      {/* <button className="nav_button__trello">Crello</button>
```
Comment (Taneros) [inline src/components/Header/index.jsx:14]: remove comments

**Comment by Taneros** [comment]
need to pull changes from last PR on main... then create new PR... otherwise do not merge...

**Pull request closed without merging**